docs/userguide/index-sc.md

```
@@ -8,12 +8,13 @@ Welcome aboard! This guide will walk you through everything this BBS has to offe
-3. [Netmail: Private Messages](#netmail-private-messages)
-4. [Doors: Games & Applications](#doors-games-applications)
-5. [File Areas](#file-areas)
-6. [Exploring the BBS Networks](#exploring-the-bbs-networks)
-7. [Your Profile & Settings](#your-profile-settings)
-8. [Getting Help](#getting-help)
+3. [Message Reader](#message-reader)
+4. [Netmail: Private Messages](#netmail-private-messages)
+5. [Doors: Games & Applications](#doors-games-applications)
+6. [File Areas](#file-areas)
+7. [Exploring the BBS Networks](#exploring-the-bbs-networks)
+8. [Your Profile & Settings](#your-profile-settings)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)